codice/ddf
# DDF-4318 As a user, I want map coordinate display settings for MGRS and UTM/UPS formats

#### What does this PR do?
Adds support for the user to specify MGRS or UTM/UPS as the bottom-left coordinate format indication. Use the top-right `Settings` dropdown to make the selection. The data is stored in user preferences. 

#### Who is reviewing it? 
@Variadicism 
@samuelechu 
@BernardIgiri 
@Dimitry-Sherman

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining 
@djblue

#### How should this be tested?
Hover the mouse around the maps and ensure the MGRS and UTM/UPS settings yield accurate coordinate reports on the bottom left, according to where the mouse is positioned. Ensure the transition is smooth and that no related issues / errors are found in the web console. 

Verify that the MGRS setting does not actually draw any coordinates when the mouse is hovered by the poles. 

#### What are the relevant tickets?
[DDF-4318](https://codice.atlassian.net/browse/DDF-4318)

#### Screenshots
See here the new settings options, as well as the coordinate strings being printed on the bottom left. 

![screen shot 2018-11-07 at 7 29 08 pm](https://user-images.githubusercontent.com/7215534/48174620-f6b08b80-e2c5-11e8-89f7-0dfb6b3a0a19.png)

![screen shot 2018-11-07 at 7 29 34 pm](https://user-images.githubusercontent.com/7215534/48174636-0af48880-e2c6-11e8-9bd5-ae5ad7fc67d9.png)

![screen shot 2018-11-07 at 7 29 53 pm](https://user-images.githubusercontent.com/7215534/48174640-0f20a600-e2c6-11e8-910d-781427d62164.png)


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
